addons/website_links/static/src/js/website_links_code_editor.js

```diff
@@ -44,35 +44,46 @@ publicWidget.registry.websiteLinksCodeEditor = publicWidget.Widget.extend({
      * @param {String} newCode
      */
     _showNewCode: function (newCode) {
-        $('.o_website_links_code_error').html('');
-        $('.o_website_links_code_error').hide();
+        // Clear error message and hide error element
+        const errorElement = this.el.querySelector(".o_website_links_code_error");
+        errorElement.innerHTML = "";
+        errorElement.style.display = "none";
```

Comment:
As we discussed previously we will use class ```d-none``` for hide/show feature.

Reply:
NOted!